Cargo.toml

```
@@ -49,7 +49,7 @@ jubjub = "0.10"
 
 # - Orchard
 nonempty = "0.7"
-orchard = { version = "0.6", default-features = false }
+orchard = { version = "0.6.1", default-features = false }
```

This PR will be blocked on @nuttycom's other PRs.
```suggestion
orchard = { version = "0.7", default-features = false }
```